src/main/proto/cosmas.proto

```diff
@@ -21,6 +21,7 @@ option java_outer_classname = "CosmasProto";
 message FileVersionInfo {
     int64 generation = 1;
     int64 timestamp = 2;
+    string actualFileId = 3;

```

Comment:
Do you really need this amendment? I think `fileId` is okay, short and matching `GetVersionRequest::fileId`. If you really want to amend "file id", I suggest "effective", in a sense similar to "effective user id" in Unix systems: id which is not your real one, but on whose behalf you take some action.

Reply:
ok, I used just `fileId`